common/go-utils/Cargo.toml

```diff
@@ -13,3 +13,6 @@ serde = "1"
 thiserror = "1"
 toml = "0.8"
 ureq = { version = "2", features = ["json"] }
+
+[dev-dependencies]
+serde = { version = "1", features = ["derive"] }
```

Comment:
I would move this to the main `serde` dep above. The dervice macro is used by this crate in non-test code, so listing the feature as a full dependency is correct.

(Btw the reason this only fails in VSCode when running individual tests is likely that VSCode is running it as a single crate rather than as a workspace - you can likely get the same effect by running `cargo test` inside the utils directory instead of from the repo root. The reason it works from the workspace level will be because of https://doc.rust-lang.org/cargo/reference/features.html#feature-unification)

Reply:
Oh interesting, and you're right about the tests!